backend/infrahub/config.py

```diff
@@ -890,16 +923,22 @@ class Settings(BaseSettings):
     logging: LoggingSettings = LoggingSettings()
     analytics: AnalyticsSettings = AnalyticsSettings()
     initial: InitialSettings = InitialSettings()
+    policy: PolicySettings = PolicySettings()
     security: SecuritySettings = SecuritySettings()
     storage: StorageSettings = StorageSettings()
     trace: TraceSettings = TraceSettings()
     experimental_features: ExperimentalFeaturesSettings = ExperimentalFeaturesSettings()
 
+    @property
+    def enterprise_features(self) -> list[EnterpriseFeatures]:
+        """Returns a list of enterprise features that are enabled based on the settings."""
+        return self.policy.enterprise_features
```

Comment:
Follow up to your comment
> I'm not sure if we need the "enterprise_features" flag or if we should instead just indicate the installation type "community" or "enterprise".

I feel indicating the installation type is enough, and our docs should clearly indicate which features are enterprise only, with their detailed behaviors